woothemes/woocommerce-rest-api-docs
# Add products and product stats analytics reports to API doc

Part of #248. This PR intends to merge into a feature branch [feature/add-wc-analytics-reports-api-doc](https://github.com/woocommerce/woocommerce-rest-api-docs/tree/feature/add-wc-analytics-reports-api-doc).

This PR adds products and product stats APIs to the analytics doc:

- Add a `wc-analytics` link in `wc/v3` introduction.
- Add introduction section
- Add authentication section
- Add index section
- Add product reports section (`/wc-analytics/reports/products`)
- Add product stats reports section (`/wc-analytics/reports/products/stats`)

@woocommerce/proton Can you help on reviewing the structure of the API doc? The content will be reviewed by @woocommerce/automata. Thank you.

#### Detailed test instruction

1. Build the doc by running `./build.sh`
2. Open `build/index.html`
3. Click `Analytics docs`
4. Verify the structure and the content in the Analytics API doc.
